airqo-platform/AirQo-frontend
# [NetManager] Analytics Refactoring

#### Summary of Changes (What does this PR do?)
- Add averages and exceedances charts.
- Enable frequent hitting of cached data by rounding dates
- Add custom chart loader
- Truncate long chart subheader

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Set this backend [PR](https://github.com/airqo-platform/AirQo-api/pull/467)
* Fetch and checkout to this branch locally
* Set the `REACT_APP_BASE_ANALYTICS_URL` in frontend `.env`. sample value `http://localhost:5000/api/v1/` assuming nothing changed
* Install requirements `npm install`
* Start netmanager application `npm run stage-mac` (macOS) or `npm run stage-pc` (windows Platform)
* Confirm the summary of changes listed above

#### What are the relevant tickets?
- [PLAT-671](https://airqoteam.atlassian.net/browse/PLAT-671)

#### Screenshots (optional)
